README.md

```diff
@@ -13,6 +13,8 @@ Dependency Analytics is powered by [Snyk Intel Vulnerability DB](https://snyk.io
 - Suggests a project level license, check for conflicts between dependency licences
 - AI based guidance for additional, alternative dependencies
 
+> **Disclaimer** : Dependency Analytics does not analyse dev/test dependencies associated with the Stack.
```

Comment:
**Disclaimer**: Dependency Analytics does not analyse **dev/test** dependencies associated with the Stack.